internal/cmd/gocdk/biome_test.go

```diff
@@ -76,14 +76,6 @@ func TestBiomeAdd(t *testing.T) {
 }
 
 func TestBiomeApply(t *testing.T) {
-	// TODO(#1809): test cases
```

Comment:
I believe these tests are all covered by the CLI test now, although a couple aren't needed anymore because we just always run `terraform init`.